### PR TITLE
[TASK] Add rollback command

### LINKED ISSUE #96: Add rollback command task
Currently there is no command for a rollback. The only rollback support is **during** deployment, but not afterwards. When the site is live, it's a pain to revert the symlinks by hand.

See Rollback in deployer: https://github.com/deployphp/deployer/blob/master/recipe/deploy/rollback.php

### PR DESCRIPTION
**Pull request by sabbelasichon**
- Fixes #96

- [x] Tests for the changes have been added (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This is a new Feature

* **What is the current behavior?** (You can also link to an open issue here)
At the moment no rollback command exists

* **What is the new behavior (if this is a feature change)?**
We now could rollback to the previous release by executing the surf rollback deployment

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No breaking change

### CODE REVIEW

File: src/Domain/Model/RollbackWorkflow.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<?php
+
+
```
Comment (simonschaufi) [inline src/Domain/Model/RollbackWorkflow.php]: one empty line too much

File: src/Task/Generic/RollbackTask.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+
```
Comment (simonschaufi) [inline src/Task/Generic/RollbackTask.php]: one empty line too much

**Comment by kaystrobach** [comment]
Imho it’s ok now

**Comment by sabbelasichon** [comment]
@kaystrobach Could you please approve this PR. Afterwards i am going to merge into master.

**Pull request merged**